test/stab.c

```diff
@@ -1466,10 +1466,9 @@ main(void)
     puts("All symbol table tests passed.");
 
     /* Cleanup */
-    if (GetTestCleanup()) {
```

Comment:
This test wasn't integrated with testframe.c and never changed the cleanup status; `GetTestCleanup()` calls weren't useful here